examples/demo_exponentiated_gradient_reduction.ipynb

```diff
@@ -4,6 +4,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://github.com/Trusted-AI/AIF360/blob/master/examples/demo_exponentiated_gradient_reduction.ipynb)\n",
```

Comment:
Looks good.

Reply:
Thank you Gowri! :)

Reply:
The hyperlink here points to the notebook in this repository. The idea of using the badge would be to link it to a colab notebook/asset so it would look something like (`https://colab.research.google.com/<path to notebook>`)